include/swift/AST/DiagnosticsSema.def

```diff
@@ -2069,6 +2069,14 @@ NOTE(codable_codingkeys_type_is_not_an_enum_here,none,
      "cannot automatically synthesize %0 because 'CodingKeys' is not an enum", (Type))
 NOTE(codable_codingkeys_type_does_not_conform_here,none,
      "cannot automatically synthesize %0 because 'CodingKeys' does not conform to CodingKey", (Type))
+NOTE(decodable_no_super_init_here,none,
+     "cannot automatically synthesize %0 because superclass does not have a callable 'init()' or 'init(from:)'", (DeclName))
```

Comment:
I had this show the specific member directly in an earlier version of this, but wanted to highlight that you can avoid this issue by either making `super` `Decodable`, or provide an `init()` (regardless of which case you were currently in). Do we prefer to just show the one we need?